tardis/energy_input/gamma_packet_loop.py

```diff
@@ -19,9 +19,11 @@
     doppler_factor_3d,
     get_index,
 )
+from tardis.opacities.compton_opacity_calculation import (
```

Comment:
Should here be "tardis.opacities.opacities" instead?

Reply:
done